.github/workflows/publish.yml

```diff
@@ -0,0 +1,26 @@
+name: Publish
+
+on:
+    release:
+        types:
+            - created
```

Comment:
We can also bind it to a tag push, up to you!
Also, we can add the `workflow_dispatch` if we need to trigger it manually